The job **`x86_64-gnu-tools`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/10752642914/job/29821184501) [(plain)](https://github.com/rust-lang/rust/commit/2f6307d1cca53c153bddee83f4331fe979c61fe4/checks/29821184501/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
  x86_64-pc-windows-msvc)
    # Strangely, Linux targets do not work here. cargo always says
    # "error: cannot produce cdylib for ... as the target ... does not support these crate types".
    # Only run "pass" tests, which is quite a bit faster.
    #FIXME: Re-enable this once CI issues are fixed
    #python3 "$X_PY" test --stage 2 src/tools/miri --target aarch64-apple-darwin --test-args pass
    #python3 "$X_PY" test --stage 2 src/tools/miri --target i686-pc-windows-gnu --test-args pass
  *)
    echo "FATAL: unexpected host $HOST_TARGET"
    exit 1
    ;;
    ;;
esac
# Also smoke-test `x.py miri`. This doesn't run any actual tests (that would take too long),
# but it ensures that the crates build properly when tested with Miri.

#FIXME: Re-enable this for msvc once CI issues are fixed
if [ "$HOST_TARGET" != "x86_64-pc-windows-msvc" ]
  python3 "$X_PY" miri --stage 2 library/alloc --test-args notest
  python3 "$X_PY" miri --stage 2 library/std --test-args notest
fi
#!/bin/sh
---

 finished in 44.807 seconds
##[endgroup]
Build completed successfully in 0:01:20
/tmp/checktools.sh: 78: Syntax error: "fi" unexpected (expecting "then")
  network time: Sat, 07 Sep 2024 16:34:28 GMT
##[error]Process completed with exit code 2.
Post job cleanup.
```

</details>

--- a/src/ci/docker/host-x86_64/x86_64-gnu-tools/checktools.sh
+++ b/src/ci/docker/host-x86_64/x86_64-gnu-tools/checktools.sh
@@ -71,7 +71,7 @@ esac
 # but it ensures that the crates build properly when tested with Miri.
 
 #FIXME: Re-enable this for msvc once CI issues are fixed
-if [ "$HOST_TARGET" != "x86_64-pc-windows-msvc" ]
+if [ "$HOST_TARGET" != "x86_64-pc-windows-msvc" ]; then
   python3 "$X_PY" miri --stage 2 library/core --test-args notest
   python3 "$X_PY" miri --stage 2 library/alloc --test-args notest
   python3 "$X_PY" miri --stage 2 library/std --test-args notest

